src/control/controlbehavior.cpp

```diff
@@ -93,13 +103,23 @@ double ControlPotmeterBehavior::valueToMidiParameter(double dValue) {
     }
 }
 
+double ControlPotmeterBehavior::scaleStartParameter() {
+    return m_dScaleStartParameter;
+}
+
+void ControlPotmeterBehavior::setScaleStartParameter(double value) {
+    // TODO(Ferran Pujol): Check if value is between valid bounds (0,1)?
```

Comment:
Wow I forgot that. I think that's the correct place. Maybe clamping the value to 0..1 and printing a warning to the log is the right thing to do. Clearly setting this outside 0..1 is a programmer error but putting an assert for this seems excessive to me. What do you think?